src/data/set_like.lean

```diff
@@ -101,9 +101,13 @@ lemma le_def {S T : A} : S ≤ T ↔ ∀ ⦃x : B⦄, x ∈ S → x ∈ T := iff
 @[simp, norm_cast]
 lemma coe_subset_coe {S T : A} : (S : set B) ⊆ T ↔ S ≤ T := iff.rfl
 
+lemma coe_mono : monotone (coe : A → set B) := λ a b, coe_subset_coe.mpr
```

Comment:
Do you want to tag this `@[mono]`?

Reply:
Yeah, once #7017 is merged

Reply:
Alright, done; I moved some imports around to be on the safe side and prevent `set_like` from importing `algebra.ordered_ring` transitively.